app/features/conference/components/Conference.js

```diff
@@ -201,7 +202,11 @@ class Conference extends Component<Props, State> {
         const roomName = url.pathname.split('/').pop();
         const host = this._conference.serverURL.replace(/https?:\/\//, '');
         const searchParameters = Object.fromEntries(url.searchParams);
-        const urlParameters = Object.keys(searchParameters).length ? searchParameters : {};
+        const locale = { lng: i18n.language };
+        const urlParameters = Object.keys(searchParameters).length ? {
```

Comment:
Please drop the check from Object.keys, Object.fromEntries will always return an object, so we can destructure it even if empty.

Reply:
True, sorry for that